server/src/db/repositories/complianceRepository.ts

```diff
@@ -212,6 +214,28 @@ const mongoComplianceRepository: ComplianceDbRepository = {
     }
   },
 
+  async createOrSubmitForm(draftData: Partial<ComplianceReport>): Promise<string> {
+    try {
+
+      const isDraft = draftData.status == StatusEnum.DRAFT;
+      const link = isDraft ? uuidv4() : '';
```

Comment:
it' not a link, it's a uuidv4 identifier. If you'd like to provide a link it should be formed as such. I don't see issues with both appraoches: providing the link that is removed upon expiration or using the identifier provided by mongo as a way to retreive drafts if createTime<expireTime (i lean towards first solution), but we should be specific with names and content.

Reply:
I will chage it to "uniqueId"